app/views/uploadHasCompleted.scala.html

```diff
@@ -0,0 +1,19 @@
+@import java.util.UUID
+@(consignmentId: UUID)(implicit request: RequestHeader, messages: Messages)
+
+@main("Upload your records") {
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      @progressIndicator(Messages("upload.progress"))
```

Comment:
Should the progress bar be here? I wouldn't have expected to see it here, but I'm not sure of the design.

Reply:
Oh no, this is calling the [progress indicator ](https://github.com/nationalarchives/tdr-transfer-frontend/blob/master/app/views/progressIndicator.scala.html) file, the one that shows which step you are on out of N steps

Reply:
You're quite right. Ignore me. 🙂